bps/bps_ApPipeMultiTractFakes.yaml

```diff
@@ -0,0 +1,44 @@
+# UPDATE THIS to the path to the pipeline to run
+# (You probably want to run a camera-specific ApPipeMultiTractFakes, not the generic one)
+pipelineYaml: '${AP_PIPE_DIR}/pipelines/ApPipeMultiTractFakes.yaml'
+
+# Just names, but UPDATE THIS AND KEEP THEM SHORT
+project: ApPipeMultiTractFakes
+campaign: DM-xxxxx
+
+submitPath: ${PWD}/bps/{outputRun}
+computeSite: ncsapool
+# Memory allocated for each quantum, in MB (if unspecified, default is 2048)
+requestMemory: 4096
+includeConfigs:
+  - ${CTRL_BPS_DIR}/python/lsst/ctrl/bps/etc/bps_defaults.yaml
+
+# UPDATE THIS set of arguments to mirror the ones you use with command-line pipetask run
+payload:
+  payloadName: DM-xxxxx-example
+  butlerConfig: /repo/main/butler.yaml
+  inCollection: DECam/defaults/hits2015,DECam/calib,u/mrawls/DM-30703-hits-crosstalk-sources,u/mrawls/DM-30703-HiTS2014-templates
+  dataQuery: "instrument='DECam' AND skymap='hsc_rings_v1' AND band IN ('g') AND exposure>410914 AND exposure<421607 AND detector NOT IN (2, 61)"
```

Comment:
Not sure, but I don't think it hurts to specify it by default in case I switch to specifying tracts/patches, since I know it's needed then.

Reply:
I'm not so sure -- my work on DM-31681 (done after my review here) suggests that the query evaluation can be _extremely_ inefficient, sometimes worse than just processing everything. In this case I think the `skymap` clause might actually hurt you because it's pulling in an unrelated dimension that needs to be matched to everything else.